<launch>
    <!--
    Provide an object name to track a specific object,
    or 'all' to track every object in the config
    file. You can pass an argument when you invoke roslaunch.
    -->
    <arg name="object_name" default="all"/>

    <arg name="config_file_name"
         default="multiobject_with_dope"
    />

    <node name="diffdope_refiner"
          pkg="diffdope_ros"
          type="refine_continuous.py"
          output="screen"
          args="$(arg config_file_name) $(arg object_name)"
    />
</launch>
